src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -588,77 +588,115 @@ ucs_status_t uct_dc_mlx5_ep_get_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov,
     return UCS_INPROGRESS;
 }
 
-ucs_status_t uct_dc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
-                                  uct_completion_t *comp)
+static UCS_F_ALWAYS_INLINE ucs_status_t
+uct_dc_mlx5_ep_flush_cancel(uct_dc_mlx5_ep_t *ep, uint16_t *sn_p)
 {
```

Comment:
i'd not make it inlined as it is not a fast path (maybe even specify noinline)

Reply:
done